spring-cloud-gcp-docs/src/main/asciidoc/spanner.adoc

```diff
@@ -794,6 +800,36 @@ List<Trade> fetchByActionNamedQuery(String action);
 
 This allows table names evaluated with SpEL to be used in custom queries.
 
+==== Projections
+
+Spring Data Spanner supports projections. You can define projection interfaces based
```

Comment:
link to `https://docs.spring.io/spring-data/commons/docs/current/reference/html/#projections` - please note we have an asciidoc variable already for the spring data commons current docs root.